### PR TITLE
[Release] Increase node memory by 2X in many_ppo test.

### PR DESCRIPTION
**Pull request by xwjiang2010**
## Related issue number

#19309

## Checks

- [X] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Review by rkooo567 — APPROVED** [review]

File: release/long_running_tests/long_running_tests.yaml [hunk 1]
```diff
@@ -78,11 +78,11 @@
...
   cluster:
     app_config: app_config.yaml
-    compute_template: tpl_cpu_3.yaml
+    compute_template: many_ppo.yaml
 
   run:
     timeout: 86400
-    prepare: python wait_cluster.py 3 600
+    prepare: python wait_cluster.py 1 600
     script: python workloads/many_ppo.py
```
Comment (rkooo567) [inline release/long_running_tests/long_running_tests.yaml:86]: Why don’t we just keep 3 nodes?

**Comment by xwjiang2010** [comment]
The test only needs one node. I don't know why we start 3 nodes in the first place.

**Comment by rkooo567** [comment]
Does that mean one node has sufficient resources to run a test, and worker nodes weren’t used? I think it’d be great to have multi node long running test because that can give stress to ray in a different way.

**Comment by xwjiang2010** [comment]
Sang, I think you brought up a good point; however, I believe the original intention was to use one node and multiple processes: https://sourcegraph.com/github.com/ray-project/ray/-/blob/release/long_running_tests/workloads/many_ppo.py?L14

**Pull request merged**